modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip.i

```diff
@@ -0,0 +1,180 @@
+mu=1.1
+rho=1.1
+advected_interp_method='average'
+velocity_interp_method='rc'
+force_boundary_execution=true
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 10
+    ymin = -1
+    ymax = 1
+    nx = 100
+    ny = 20
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  fv_bcs_integrity_check = true
+[]
+
+[Variables]
+  [u]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+    initial_condition = 1
+  []
+  [v]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+    initial_condition = 1
+  []
+  [pressure]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+  []
+[]
+
+[FVKernels]
+  [mass]
+    type = NSFVKernel
```

Comment:
If we have `NSFVMassKernel`, then it will only remove on parameter, the `advected_quantity` parameter. The other parameters should be set by the user in the input file and they should think about what they are doing.

The verbosity of these inputs would be significantly reduced by adding action syntax, which I've opened an issue for: https://github.com/idaholab/moose/issues/16283

This PR is already really long though

Reply:
I think I can add `NSFVMassKernel` though. I have already renamed `NSFVKernel` to `NSFVAdvectionKernel`

Reply:
Now I will be penalized for writing lots of tests :laughing: :cry: Lots of input files to change

Reply:
So I'm hoping that the new naming scheme will make this look a little less daunting for someone who is familiar with our previous NS objects. However, as I said before I think the action is the thing that is really going to help with these verbose input files.